tests/integration/linux_bridge_test.py

```diff
@@ -120,6 +120,21 @@ def bond0(port0_up):
         yield bond0
 
 
+@pytest.fixture
+def bridge_with_port_vlan_config(port0_up):
+    access_port_state = TestVlanFiltering._generate_vlan_filtering_config(
+        LinuxBridge.Port.Vlan.Mode.TRUNK,
+        native_vlan=True,
+        tag=TestVlanFiltering.access_tag,
+        trunk_tags=TestVlanFiltering._generate_vlan_id_config(100, 101, 102),
+    )
+    port_name = port0_up[Interface.KEY][0][Interface.NAME]
+    bridge_state = _create_bridge_subtree_config((port_name,))
+    bridge_state[LinuxBridge.PORT_SUBTREE][0].update(access_port_state)
+    with linux_bridge(TEST_BRIDGE0, bridge_state):
+        yield
+
+
```

Comment:
I think we are also following "caller before the callee" in integration tests. So please, place this at the buttom. @EdDev What do you think? Does it apply also to fixtures?

Reply:
AFAIU, fixtures go at top of the tests. Let's way for @EdDev 's say on this.

Reply:
Looks good, thanks!